grouper/models.py

```diff
@@ -652,7 +653,9 @@ class UserToken(Model):
     created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
     disabled_at = Column(DateTime, default=None, nullable=True)
 
-    secret = Column(String(length=32), default=_make_secret, unique=True, nullable=False)
+    hashed_secret = Column('secret', String(length=64), unique=True, nullable=False)
+
+    secret = None
```

Comment:
this is a little confusing because most (all?) attributes of models in grouper are actual columns in the db. can we maybe make this a private attribute and hide the accessor behind a property function?

alternative solution, and maybe much cleaner, is to have the `add` method return both the model and the unhash secret in the return. that keeps the contract (attributes are columns) and makes it explicit the unhashed secret is ephemeral.